recipes/xgeo/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set version = "1.0" %}
+
+package:
+  name: xgeo
+  version: {{ version }}
+
+source:
+  url: https://github.com/Geosynopsis/xgeo/archive/xgeo-v{{ version }}.tar.gz
+  sha256: 2b494a56c8f91967828c49806a0e8cdde2f4af8c24bc2b2ede509ce6ff96a7dc
+
+build:
+  number: 0
+  script:  "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [(py<36)]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```

If it becomes noarch you can keep the Python version constraint with:
```suggestion
    - python >=3.6
```